docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>looks like the plot here is missing?</p><p> </p><p>Should also add some text after explaining what info we get from this plot and how its used later (ie, is it how you decided what to use for num_pitch etc below?)</p><p> </p><p>might also be useful to do it at deliberately too low resolution to show what a bad example looks like</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>